### PR TITLE
Add icon set

### PR DESCRIPTION
**Pull request by Vadim-x-squad**
Added icon package "react-native-icomoon", and exported all icons from figma to repository

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,7467 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (valentinmagrez) [inline yarn.lock:1]: Right now we are not using yarn but only npm in our library., if you can only commit package.json and package-lock

File: package.json [hunk 2]
```diff
@@ -80,5 +80,8 @@
       "<rootDir>/example/node_modules",
       "<rootDir>/lib/"
     ]
+  },
+  "dependencies": {
+    "react-native-icomoon": "^0.1.1"
```
Comment (valentinmagrez) [inline package.json:85]: I know that for some library like react-native-paper (often library called 'react-native-something') that has some native code that need to be link, we needed to install it as a peerdependencies. Dont know if it the case here with react-native-icomoon

File: yarn.lock (hunk 1 above)
Comment (Vadim-x-squad) [reply yarn.lock]: deleted

**Review by valentinmagrez — APPROVED** [review]

File: src/components/icons/IconProps.ts [hunk 3]
```diff
@@ -1,7 +1,84 @@
...
+    | 'shape-27'
+    | 'shape-28'
+    | 'shape-29'
+    | 'shape-30'
+    | 'shape-31'
+    | 'shape-32'
+    | 'shape-33'
+    | 'shape-34'
+    | 'shape-35'
+    | 'shape-36'
```
Comment (GoMino) [inline src/components/icons/IconProps.ts:72]: can we have more explicit names for these shapes ?

**Review by GoMino — APPROVED** [review]

**Pull request merged**